packages/core-api/lib/versions/2/handlers/transactions.js

```diff
@@ -40,12 +40,17 @@ exports.store = {
    * @return {Hapi.Response}
    */
   async handler (request, h) {
-    if (!transactionPool) {
+    const blockchain = container.resolvePlugin('blockchain')
+    if (!transactionPool || !blockchain) {
       return {
         data: []
       }
     }
 
+    if (request.payload.transactions.length > transactionPool.options.maxTransactionsPerRequest) {
```

Comment:
This is a v2 endpoint, we are not using those ghetto v1 solutions here.

You will have to update the joi schema to be limited to the configured value.

https://github.com/ArkEcosystem/core/blob/develop/packages/core-api/lib/versions/2/schema/transactions.js#L18